pydatalab/pydatalab/routes/v0_1/users.py

```diff
@@ -0,0 +1,34 @@
+from bson import ObjectId
+from flask import Blueprint, jsonify, request
+from flask_login import current_user
+
+from pydatalab.mongo import flask_mongo
+
+user = Blueprint("users", __name__)
+
+
+@user.route("/users/<user_id>", methods=["PATCH"])
+def save_user(user_id):
+    request_json = request.get_json()
+
+    display_name = request_json.get("display_name")
+    contact_email = request_json.get("contact_email")
+
+    if not current_user.is_authenticated:
+        return jsonify(status="error"), 401
+
+    if current_user.id != user_id and current_user.role != "admin":
+        return jsonify(status="error"), 403
```

Comment:
Now that the `Person` model has additional validation, we need to make sure the data is piped through the model here before updating the database. One way to do that would be to pull the existing user data from the db, update it with the new data, then create a model. If it passes the model, then update the db.

Alternatively, we could just individually validate the two fields considered here by defining specialist types for email addresses and display names, then use those in the `Person` model directly (see other comment)